tilelang/language/tir/ir.py

```
@@ -169,6 +172,25 @@ def wrapped(*args, **kwargs):
 
 
+def Select(condition: PrimExpr | bool, true_value: PyPrimExpr, false_value: PyPrimExpr, span=None) -> PrimExpr:
+    """Construct a TIR select expression.
+
+    ``Select`` evaluates both value expressions. Use ``if_then_else`` when a
+    branch must guard side effects or out-of-bounds memory accesses.
+    """
+    return _Select(condition, true_value, false_value, span)
+
+
+def min(a: PrimExpr, b: PrimExpr) -> PrimExpr:  # pylint: disable=redefined-builtin
```

Ruff (0.15.21)

[error] Variable `min` is shadowing a Python builtin

(A001)